packages/astro/test/dev-routing.test.js

```diff
@@ -113,9 +113,9 @@ describe('Development Routing', () => {
 			expect(response.status).to.equal(200);
 		});
 
-		it('404 when loading subpath root without trailing slash', async () => {
+		it('200 when loading subpath root without trailing slash', async () => {
 			const response = await fixture.fetch('/blog');
-			expect(response.status).to.equal(404);
+			expect(response.status).to.equal(200);
 		});
```

Comment:
Had to update some tests behaviours when fetching a site with `base` without trailing slash. When you have `base: '/blog'` and `trailingSlash: 'ignore'`, fetching `http://mysite.com/docs` (no trailing slash) will now work.